pages/repo/edit.js

```diff
@@ -639,15 +684,13 @@ export class EditorPage extends Component {
     Router.pushRoute('repo/raw', {
       repoId: repoId.split('/'),
       commitId,
-      ...(commitId === 'new'
-        ? { template: this.props.router.query.template }
-        : {})
+      ...(commitId === 'new' ? { schema: this.props.router.query.schema } : {})
```

Comment:
```suggestion
      ...(commitId === 'new' ? { schema: this.props.router.query.schema || this.props.router.query.template } : {})
```

Reply:
also the raw page needs to be adapted to support `query.schema` (also with a fallback)